client/src/components/QandA.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import SearchBar from './QandA/SearchBar.jsx';
+import QuestionsList from './QandA/QuestionsList.jsx';
+
+const QandA = (props) => {
+
+  return (
+    <div>
+      <p>Questions & Answers</p>
+      <SearchBar />
+      <QuestionsList product_id={40356}/>
```

Comment:
so this is hard-coded right now to take a product id that will be used to render the rest of the components, just making not of this so that we know to pass Q&A the proper data from App.jsx

Reply:
Yes, it is currently hard-coded. Will be updated when Overview and other modules come together.